pkg/gktest/filter.go

```diff
@@ -1,58 +1,59 @@
 package gktest
 
-// Filter filters suites and tests to run.
+// Filter filters constraints and tests to run.
```

Comment:
It looks like the design doc has changed a bit.

Should suites still have a name, and should users be able to filter off them (think evaluating a test dirtree via `/...`)?

Is constraint name considered to be part of the test identity? What happens if we start introducing mutation-only tests? Should we only filter off of test name to avoid that situation?

Reply:
Modified names per meeting. Going to call them Suites/Tests/Cases.

Reply:
We are still coupling the identity of a test/case to the constraint name, by virtue of it being the thing against which users are filtering tests.

What happens if we add mutation and we have a test case that does not use a constraint? How will we uniquely identify those tests.

I suppose you could say: "the mutator name"... but if both a constraint and a mutator are provided?

Reply:
Suites define `metadata.name`. Tests define `.name`, and Cases define `.name`.